src/components/ConnectModal.tsx

```diff
@@ -11,6 +11,29 @@ interface ConnectModalProps {
   closeModal: () => void;
 }
 
+const connectEmail = async () => {
```

Comment:
Can we turn this into a proper function and then hardcode the parameters somewhere else instead if it's necessary?

Reply:
Yes, as I said, it's just barebones. I need it merged in so that I can pick up the two PRs that were put up since I checked this branch out.